driver-mapping/src/main/java/com/datastax/driver/mapping/QueryType.java

```diff
@@ -64,14 +64,21 @@ String makePreparedQueryString(TableMetadata table, EntityMapper<?> mapper) {
                                   ? insertInto(mapper.getKeyspace(), mapper.getTable())
                                   : insertInto(table);
                     for (ColumnMapper<?> cm : mapper.allColumns())
-                        insert.value(cm.getColumnName(), bindMarker());
+                        if (cm.kind != ColumnMapper.Kind.COMPUTED)
+                            insert.value(cm.getColumnName(), bindMarker());
                     return insert.toString();
                 }
             case GET:
                 {
+                    ArrayList<String> columns = new ArrayList<String>();
+                    for (ColumnMapper cm : mapper.allColumns()){
+                        columns.add(cm.getColumnName());
+                    }
+                    String[] columnsArray = new String[columns.size()];
+                    columnsArray = columns.toArray(columnsArray);
                     Select select = table == null
-                                  ? select().all().from(mapper.getKeyspace(), mapper.getTable())
-                                  : select().all().from(table);
+                                  ? select(true, columnsArray).from(mapper.getKeyspace(), mapper.getTable())
+                                  : select(true, columnsArray).from(table);
```

Comment:
I'd rather implement this with a new `select().raw(String)`, that would work in a similar way to the existing `select().fcall(...)`. I think this would have less overall impact on the query builder code.
What you would do here is initialize the `select()`, then chain the `raw()` calls with a for loop on the columns, and finally add the `from` after the loop.